base/threadingconstructs.jl

```diff
@@ -232,35 +272,63 @@ macro threads(args...)
 end
 
 """
-    Threads.@spawn expr
+    Threads.@spawn [:default|:interactive] expr
```

Comment:
The name `:interactive` seems to be rather too specific. I think it's reasonable to put any latency-oriented tasks in the "secondary" task pool. Not sure what the right name is, though. Maybe something like `:io`, `:shortlatency`, `:quick`, `:I_promise_that_I_yield_often`, ...

Reply:
I find `:interactive` more appropriate than any of those but `:shortlatency` (or maybe `:lowlatency`) is the best of those.I'm open to other suggestions.

The fact is that there is really no difference between the threadpools -- the term `:interactive` is actually a suggestion to the programmer to abide by a handshake agreement to avoid running long duration tasks in this pool, or to yield frequently.

Reply:
I personally prefer `:io` but `:lowlatency` also sounds good to me.

I don't think `:interactive` is good. Maybe you have some sensory devices and do some quick managing before sending it to the main pool.

Reply:
`:io` is nicely minimal, but incomplete? How about we get some more input? @vtjnash, @JeffBezanson, @jpsamaroo, thoughts?

Reply:
`:interactive` or `:lowlatency` are my votes. `:io` is an acronym which may not be universally understood, and `:shortlatency` is really better phrased as `:lowlatency`. `:quick` is a super overloaded word and not particularly useful.